presto-main/src/main/java/com/facebook/presto/sql/planner/iterative/rule/RewriteAggregationIfToFilter.java

```diff
@@ -74,7 +77,10 @@
  * <p>
  * Note that unwrapping the IF expression in the aggregate might cause issues if the true branch return errors for rows not matching the filters. For example:
  * 'IF(CARDINALITY(array) > 0, array[1]))'
- * Session property AGGREGATION_IF_TO_FILTER_REWRITE_STRATEGY and canUnwrapIf() control whether to enable IF unwrapping.
+ * Session property AGGREGATION_IF_TO_FILTER_REWRITE_STRATEGY and canUnwrapIf() control whether to enable IF unwrapping:
+ * 1. If the strategy is FILTER_WITH_IF, then keep the IF expression.
+ * 2. If the strategy is UNWRAP_IF_SAFE/UNWRAP_IF and the IF expression can be unwrapped safely, then do so.
```

Comment:
UNWRAP_IF doesn't guarantee that the IF can be unwrapped safely. We can remove it from this bullet point.